src/lib/UI/Config/Mapper/FieldType/RichText/CustomTag.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
```

Comment:
Add `declare(strict_types=1);` here and in other PHP files.

Reply:
> Add `declare(strict_types=1);` here and in other PHP files.

Done.